curriculum/challenges/english/blocks/workshop-game-settings-panel/68e9a834dfef752dc35f6f90.md

```diff
@@ -0,0 +1,249 @@
+---
+id: 68e9a834dfef752dc35f6f90
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
```

Comment:
this is a lot of text to read
<img width="649" height="574" alt="Screenshot 2025-10-16 at 12 22 16 PM" src="https://github.com/user-attachments/assets/c9bb23e7-148e-4aca-a145-d1bdf8c306d1" />


It would be better to break this step up into two steps

The new last step could start with the bit of instruction `Set the display to block and text-align to center.`